### DISCUSSION TITLE
the bevy will break down after use feature

### DISCUSSION THREAD

**Discussion by Tom-Jim** [bevyengine/bevy#13730] [Q&A] [answered]
when i use the feature like this ,the program will break,and report it reached a unreachable code,but when use nothing it will run normally ,how can i fix it? 
![Screenshot 2024-06-07 103835](https://github.com/bevyengine/bevy/assets/83345710/27f991d1-5f70-4d7e-8def-5dfb7570dee8)
![Screenshot 2024-06-07 103901](https://github.com/bevyengine/bevy/assets/83345710/5c3b1dc7-91ad-46d5-a4c6-3b4709a719dd)

**Comment by Tom-Jim** [comment]
by he way,when i not use feature,it's still have chance to display nothing.so i have to try many times. why it can't show success at once?
![Screenshot 2024-06-07 104518](https://github.com/bevyengine/bevy/assets/83345710/03021448-73a6-40ce-a8a9-ad776459c267)

**Reply by bugsweeper** [reply, marked as answer]
Here is [solved issue](https://github.com/bevyengine/bevy/issues/13232#event-12937051845) with same panic on same feature

**Reply by Tom-Jim** [reply]
ok，thanks for guiding

**Comment by NyxAlexandra** [comment]
Clara :O
